src/patternfly/demos/NotificationDrawer/notification-drawer-demo.hbs

```diff
@@ -68,8 +68,8 @@
         {{/drawer-content}}
         {{#> drawer-panel drawer-panel--NoBody="true" drawer-panel--modifier="pf-m-width-33"}}
           {{#> drawer-body drawer-body--modifier="pf-m-no-padding"}}
-            {{#> drawer-demo-notification-drawer}}
-            {{/drawer-demo-notification-drawer}}
+            {{#> notification-drawer-demo-groups}}
```

Comment:
```suggestion
            {{> notification-drawer-demo-groups}}
```

You could get rid of closing tag since you're not passing in any content here.